packages/origin-backend-utils/src/guard/ActiveOrganizationGuard.ts

```diff
@@ -0,0 +1,25 @@
+import { IUser, OrganizationStatus } from '@energyweb/origin-backend-core';
+import { CanActivate, ExecutionContext, HttpException, Injectable } from '@nestjs/common';
+import { Reflector } from '@nestjs/core';
+
+@Injectable()
+export class ActiveOrganizationGuard implements CanActivate {
+    constructor(private reflector: Reflector) {}
+
+    canActivate(context: ExecutionContext): boolean {
+        const request = context.switchToHttp().getRequest();
+        const user = request.user as IUser;
+        const _user = user as IUser;
+
+        if (_user.organization?.status !== OrganizationStatus.Active) {
+            throw new HttpException(
+                `Only active organizations can perform this action. Your status is ${
+                    OrganizationStatus[_user.status]
```

Comment:
Should be `OrganizationStatus[_user.organization.status]` or just `_user.organization.status` since we use string enums.